app/models/course_enrichment.rb

```diff
@@ -41,4 +43,10 @@ class CourseEnrichment < ApplicationRecord
   def has_been_published_before?
     last_published_timestamp_utc.present?
   end
+
+  def publish(current_user)
+    update(status: 'published',
```

Comment:
:published instead?

Reply:
you mean like this?
```
    update(status: :published,
          last_published_timestamp_utc: Time.now.utc,
          updated_by_user_id: current_user.id)
```
The tests still pass with the above but I'm not inclined to change it unless there's a compelling reason, we've spent a lot of time on this PR already. If that's not what you mean please post full example code so I can see what you are suggesting.

Reply:
yes that, symbol always seems the better practice, https://stackoverflow.com/questions/255078/whats-the-difference-between-a-string-and-a-symbol-in-ruby